primitives/src/assets.rs

```diff
@@ -58,3 +58,11 @@ pub struct CustomMetadata {
 	/// The token conversion rate of Native to Foreign, ie. 1::10
 	pub conversion_rate: Option<ConversionRate>,
 }
+impl CustomMetadata {
+	pub fn convert_from_foreign_to_native(&self, fee: u128) -> Option<u128> {
+		match self.conversion_rate {
+			Some(value) => Some(fee * value.native as u128 / value.foreign as u128),
```

Comment:
Almost certainly not correct.  How is `ConversonRate` intended to be used @lreesby ?